features/support/capybara.rb

```diff
@@ -1,117 +1,49 @@
+    # Workaround https://bugs.chromium.org/p/chromedriver/issues/detail?id=2650&q=load&sort=-id&colspec=ID%20Status%20Pri%20Owner%20Summary
+    '--disable-site-isolation-trials',
+    # When width < 1200px, vertical navigation overlaps the page's main content and that will make
+    # some cucumbers fail
+    '--window-size=1200,2048',
+  ]
+}.freeze
 
 Before '@chrome' do
   Capybara.current_driver = :chrome
 end
 
-Before '@firefox' do
-  Capybara.current_driver = :firefox
-end
-
-Around '@security' do |scenario, block|
-  with_forgery_protection(&block)
-end
-
-# monkeypatch to fix
-# not opened for reading (IOError)
-# /cucumber-1.3.20/lib/cucumber/formatter/interceptor.rb:33:in `each'
-# /cucumber-1.3.20/lib/cucumber/formatter/interceptor.rb:33:in `collect'
-# /cucumber-1.3.20/lib/cucumber/formatter/interceptor.rb:33:in `method_missing'
-require 'cucumber/formatter/interceptor'
-class Cucumber::Formatter::Interceptor::Pipe
-  def is_a?(klass)
-    super || klass == IO
-  end
-end
-
-Capybara.register_driver :firefox do |app|
-  Capybara::Selenium::Driver.new(app, browser: :firefox)
-end
-
-Capybara.register_driver :headless_firefox do |app|
-  options = Selenium::WebDriver::Firefox::Options.new
-
-  options.add_argument('-headless')
-  options.add_argument('--window-size=1280,2048')
-
-  driver = Capybara::Selenium::Driver.new(app, browser: :firefox, options: options)
-
-  driver
-end
-
-Capybara.register_driver :firefox do |app|
-  options = Selenium::WebDriver::Firefox::Options.new
-
-  options.add_argument('--window-size=1280,2048')
-
-  driver = Capybara::Selenium::Driver.new(app, browser: :firefox, options: options)
-
-  driver
-end
-
+# Use this driver to debug scenarios locally by adding tag @chrome on top of the scenario
 Capybara.register_driver :chrome do |app|
-  options = Selenium::WebDriver::Chrome::Options.new
-  options.add_argument('--window-size=1280,2048')
-  options.add_argument('--disable-search-engine-choice-screen')
+  options = Selenium::WebDriver::Options.chrome(**BASE_DRIVER_OPTIONS)
+
   Capybara::Selenium::Driver.new(app, browser: :chrome, options: options)
 end
 
 Capybara.register_driver :headless_chrome do |app|
-  options = Selenium::WebDriver::Options.chrome(
-    logging_prefs: { performance: 'ALL', browser: 'ALL' },
-    perf_logging_prefs: { enableNetwork: true }
-  )
+  options = Selenium::WebDriver::Options.chrome(**BASE_DRIVER_OPTIONS)
+
+  options.logging_prefs = { performance: 'ALL', browser: 'ALL' }
+
+  options.add_option(:perf_logging_prefs, enableNetwork: true)
 
   options.add_argument('--headless=new')
   options.add_argument('--no-sandbox')
   options.add_argument('--disable-popup-blocking')
-  options.add_argument('--window-size=1280,2048')
   options.add_argument('--host-resolver-rules=MAP * ~NOTFOUND , EXCLUDE *localhost*')
-  options.add_argument('--disable-search-engine-choice-screen')
```

Comment:
Not needed anymore?

Reply:
I am not sure why this was removed here, but `--disable-site-isolation-trials` was added :shrug: 

I kept this setting in https://github.com/3scale/porta/pull/3947